src/coreclr/src/vm/comtoclrcall.h

```diff
@@ -36,7 +36,7 @@ enum ComCallFlags
     enum_IsWinRTCall                = 0x2000,   // The method is declared on a WinRT interface/delegate
     enum_IsWinRTCtor                = 0x4000,   // The method is a WinRT constructor
     enum_IsWinRTStatic              = 0x8000,   // The method is a WinRT static
```

Comment:
Are the remaining WinRT related enums still needed?

Reply:
Nope. I must have missed these on my most recent passes. I'll remove them/mark them as unused.